include/boost/geometry/strategies/geographic/distance_cross_track.hpp

```diff
@@ -328,22 +359,25 @@ private :
         bool dist_improve = true;
         CT s14 = s14_start;
 
-        do{
-            prev_distance = res34.distance;
+        do {
+            auto prev_distance = res34.distance;
+            auto prev_res = res14;
 
             // Solve the direct problem to find p4 (GEO)
             res14 = direct_distance_type::apply(lon1, lat1, s14, a12, spheroid);
 
             // Solve an inverse problem to find g4
-            // g4 is the angle between segment (p1,p2) and segment (p3,p4) that meet on p4 (GEO)
+            // g4 is the angle between segment (p1,p2) and segment (p3,p4)
+            // that meet on p4 (GEO)
 
-            CT a4 = inverse_dist_azimuth_type::apply(res14.lon2, res14.lat2,
-                                                lon2, lat2, spheroid).azimuth;
-            res34 = inverse_distance_azimuth_quantities_type::apply(res14.lon2, res14.lat2,
-                                                                    lon3, lat3, spheroid);
+            CT a4 = inverse_dist_azimuth_type
+                    ::apply(res14.lon2, res14.lat2, lon2, lat2, spheroid).azimuth;
```

Comment:
I can see three types of formatting for such call in this file. In addition to this one there are also:
```
            CT const dist_l = inverse_distance_type::apply
                (lon3, lat3, pl_lon, pl_lat, spheroid).distance;
```
and
```
                s14 -= inverse_distance_type::apply(res14.lon2, res14.lat2,
                                                    pl_lon, pl_lat,
                                                    spheroid).distance/2;
```

I don't know which one is the best but we could pick one of them preferably with the least number of lines. Do we have guidelines for this?

I also insist on not breaking lines shorter than 101 characters.

Reply:
what about this ?
```
s14 -= inverse_distance_type::apply(
    res14.lon2, res14.lat2, pl_lon, pl_lat, spheroid).distance/2;
```

Reply:
for the rest see comments above

Reply:
Whatever works for you, just be consistent. I prefer the second one with all arguments at the same place but I'm ok with a different one if you prefer it.